jobs/refresh-humci-warehouse.yml

```diff
@@ -85,6 +85,8 @@ configuration:
           - tableName: "oncology_program"
           - tableName: "oncology_diagnosis"
           - tableName: "all_medication_dispensing"
+          - tableName: "new_medication_dispensing"

```

Comment:
Why "new_medication_dispensing"?

Reply:
we have two forums for dispensing one used to be the main, and the new one, and each one has different source of data
so creating two tables and going to merge in a derived table

Reply:
Calling something "new_xyz" probably isn't the way we want to go though.  What happens when a 3rd option emerges?  Do we have "new_new_xyz"?

I'd think we'd want to have one medication dispensing table which can be adapted for both models.  If that isn't possible, just a more descriptive name.